x-pack/plugins/features/server/oss_features.ts

```diff
@@ -454,10 +454,43 @@ export const buildOSSFeatures = ({
           },
           savedObject: {
             all: [],
-            read: ['files'],
+            read: ['file', 'fileShare'],
           },
           ui: [],
-          api: ['files:manageFiles'],
+          api: ['files:manageFiles', 'files:defaultImage'],
+        },
+      },
+    },
+    {
+      id: 'filesSharedImage',
+      name: i18n.translate('xpack.features.filesSharedImagesFeatureName', {
+        defaultMessage: 'Shared images',
+      }),
+      order: 1600,
+      category: DEFAULT_APP_CATEGORIES.management,
+      app: ['kibana'],
+      catalogue: [],
+      privilegesTooltip: i18n.translate('xpack.features.filesSharedImagesPrivilegesTooltip', {
+        defaultMessage: 'Control access to the global set of images stored in Kibana',
```

Comment:
What does "the global set of images" refer to?  Can it be removed?

> Controls access to the images stored in Kibana.

This wording might fit in better with the other tooltips in this area.  (However, the tooltips aren't terribly consistent.)

> Required to access images stored in Kibana.

Note the ending period.

Reply:
> What does "the global set of images" refer to? Can it be removed?

Good point, I was hoping to convey this is a subset of images that happen to be globally accessible, not necessarily ALL images in Kibana. Some solutions may choose to control access in ways that may mean they are not part of this global set.

> Required to access images stored in Kibana.

I think this is good enough for now! Users will need _some_ permission to access images.